WcaOnRails/app/controllers/admin_controller.rb

```diff
@@ -157,14 +157,26 @@ def person_data
   end
 
   def compute_auxiliary_data
-    @reason_not_to_run = ComputeAuxiliaryData.reason_not_to_run
   end
 
   def do_compute_auxiliary_data
     ComputeAuxiliaryData.perform_later unless ComputeAuxiliaryData.in_progress?
     redirect_to admin_compute_auxiliary_data_path
   end
 
+  def generate_exports
```

Comment:
Why is there an empty method here?

Reply:
Because Rails mandates that for every route in `config/routes.rb`,  there must be a corresponding method in the corresponding controller. Basically, this method loads a page that is so plain and basic that it doesn't need any computations.